src/repositories/github_updater.rs

```diff
@@ -143,8 +101,17 @@ impl GithubUpdater {
             }),
         )?;
         if let Some(repo) = response.data.repository {
-            self.store_repository(conn, &repo)?;
-            Ok(Some(repo.id))
+            Ok(Some(self.store_repository(
+                conn,
+                Self::hosts()[0],
+                &repo.id,
+                &repo.name_with_owner,
+                &repo.description,
+                &repo.pushed_at,
+                repo.stargazer_count,
+                repo.fork_count,
+                repo.issues.total_count,
+            )?))
         } else if let Some(error) = response.errors.get(0) {
             use GraphErrorPath::*;
             match (error.error_type.as_str(), error.path.as_slice()) {
```

Comment:
when running the github backfill, I saw 
```
Error: error decoding response body: missing field `path` at line 1 column 70
```
plenty of times. 

It seems to me there is a (new?) error type being returned, while in the corresponding `GraphError`, `path` is mandatory. ( wild guess)

The repos I saw didn't exist on github, so likely it's somewhere around that specific error handling. 

(_coming from the diff it doesn't look like that part was changed, so likely this would also fail on master, and could also be a new issue / bug report_) 

Reply:
Please open an issue for this, it's clearly outside of the scope of this PR. ;)

Reply:
will do